.changeset/fluffy-windows-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ultraviolet/ui": minor
+---
+
+fix(SelectInputV2): pass string or string[] for values instead of an option (more compliant to native select tag). component is now controllable
```

Comment:
```suggestion
Fix <SelectInputV2 /> component to pass string or string[] for values instead of an option (more compliant to native select tag). component is now controllable
```